ajax/libs/open-iconic/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "open-iconic",
+  "filename": "font/css/open-iconic.min.css",
+  "description": "An open source icon set with marks in SVG, sprite, webfont and raster format",
+  "version": "1.1.1",
+  "keywords": [
+    "icon",
+    "iconic",
+    "open-iconic",
+    "svg",
+    "sprite",
+    "font",
+    "png",
+    "webp"
+  ],
+  "homepage": "https://useiconic.com/open",
+  "author": {
+    "name": "Iconic",
+    "email": "yourfriends@useiconic.com",
+    "web": "http://useiconic.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iconic/open-iconic.git"
+  },
+  "licenses": [
+    "MIT",
+    "OFL-1.1"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/iconic/open-iconic.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "font/**/*.!(less|scss|styl)",
+          "+(svg|sprite)/*.svg",
+          "+(png|webp)/*.+(png|webp)"
```

Comment:
`png` and `webp` are under different directory and they are different file extension. I don't think it's good to combine these two.